lib/komponent/engine.rb

```diff
@@ -9,6 +9,7 @@
 
 module Komponent
   class Engine < Rails::Engine
+    engine_name "komponent"
```

Comment:
I ask myself if we must do this in the applicaiton `config/routes.rb` to let user change the name if he needs to.

Reply:
Or use `komponent_engine`/`komponent_app[lication]` as name.

Reply:
`"komponent"` sounds good to me.